functions.json

```diff
@@ -1,5 +1,11 @@
 {
   "$schema": "https://docs.renovatebot.com/renovate-schema.json",
   "extends": ["github>reside-eng/renovate-config:service"],
-  "packageRules": []
+  "packageRules": [
+    {
+      "description": "Skip update to v4 of serverless since (still beta) [PLAT-7724]",
```

Comment:
Any idea when this is out of their consideration of "beta"? How beta is this exactly and what do we think the risk is here? I fear that we will add this ignore and forget to check back and update for a while.

Reply:
Here is the upgrade guide, but they don't make mention of when it is planned for being out of beta that I saw: https://www.serverless.com/framework/docs/guides/upgrading-v4